SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+There is only one version track for CISDSC so all patches will be applied to the latest version only.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| Latest  | :white_check_mark: |
+
+## Reporting a Bug or Vulnerability
+
+Critical vulnerabilities can be responsibly disclosed to [securitysupport@illinois.edu](mailto:securitysupport@illinois.edu). Bugs and low level vulneratbilities can be reported via repository issues.
```

Comment:
Have there been any discussion whether to use `securitysupport@` or `security@` here? I could see it either way.

Reply:
since security@ goes to the CSOC I would expect we would want to keep it in securitysupport@ but you or Taylor can certainly over rule me here.